<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonBar</class>
 <widget class="QWidget" name="ButtonBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>50</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>50</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>50</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>400</width>
    <height>50</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>394</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QToolButton" name="saveButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/save.png</normaloff>:/icons/icons/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="runMainScript">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/saveMain.png</normaloff>:/icons/icons/saveMain.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="runButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/executeAll.png</normaloff>:/icons/icons/executeAll.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="runPartialButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/play.png</normaloff>:/icons/icons/play.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="searchAndReplaceButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/searchReplace.png</normaloff>:/icons/icons/searchReplace.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="templateButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/template.png</normaloff>:/icons/icons/template.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearTerminalButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/clearConsole.png</normaloff>:/icons/icons/clearConsole.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QToolButton" name="clearButton">
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/icons/icons/clearEditor.png</normaloff>:/icons/icons/clearEditor.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>30</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
